modules/formulize/templates/admin/element_optionlist.html

```diff
@@ -54,5 +63,65 @@
     $("[name=cleardef]").blur();
   });
   <{/if}>
+  
+  <{if $content.type eq "select"}>
+  checkNAMES();
+  /**
+   *this function will consantly check for {USERNAMES} and {FULLNAMES}
+   */
+  function checkNAMES(){
+    var x=0,y=-1,z=-1;    //y for {FULLNAMES}  z for {USERNAMES}
+    for(x=0;x<length;x++){

```

Comment:
The way the {USERNAMES}/{FULLNAMES} feature works is that one of those strings has to appear in the very first option, and there are no other options at all. There can't be other items in the list. That's why those options have to appear in the very first text box.